### PR TITLE
Web refactor

### PR DESCRIPTION
**Pull request by alexrchies**
## TYPE
- Restyling

## Description
Apply new design to the website

### Documentation
- [ ] Have changelog.md updated

### Code
- [ ] Have 3 spaces indentation

### Other observations
- Add other information you consider important

### Post-review reminder
- Remember reviewer approve changes inside the Files Changed tab (Review changes dropdown)
- Squash commits
- Release version in case of breaking changes and notify of all front
## TYPE
- Restyling

## Description
Apply new design to the website

### Documentation
- [x] Have changelog.md updated

### Code
- [ ] Have 3 spaces indentation

### Other observations
-

### Post-review reminder
- Remember reviewer approve changes inside the Files Changed tab (Review changes dropdown)
- Squash commits
- Release version in case of breaking changes and notify of all front

### CODE REVIEW

File: src/app/layout/menu-item/menu-item.component.ts [hunk 1]
```diff
@@ -60,14 +74,23 @@ export class LayoutMenuItemComponent implements OnChanges, OnInit {
...
    getOpen(routerActive: boolean): boolean {
-      this.isOpen = routerActive;
+      console.log(this.menu.link.includes(this.activeRoute));
+
+      if (this.menu.link.includes(this.activeRoute)) {
+         this.isOpen = true;
+      } else {
+         this.isOpen = false;
+      }
+      //this.isOpen = routerActive;
```
Comment (EvaMGC) [inline src/app/layout/menu-item/menu-item.component.ts]: Remove commented line

File: src/assets/langs/es.json [hunk 2]
```diff
@@ -1,6 +1,11 @@
 {
    "MENU": {
-      "OVERVIEW": "Overview",
+      "OVERVIEW": {
```
Comment (EvaMGC) [inline src/assets/langs/es.json]: Translate to spanish

File: src/index.html [hunk 3]
```diff
@@ -3,12 +3,14 @@
    <head>
       <meta charset="UTF-8">
       <meta http-equiv="X-UA-Compatible" content="IE=edge">
-      <title>Stratio Components Library</title>
+      <title>Stratio Egeo - Documentation</title>
```
Comment (EvaMGC) [inline src/index.html]: Add to translate file

File: src/app/layout/menu-item/menu-item.component.ts [hunk 4]
```diff
@@ -60,14 +74,23 @@ export class LayoutMenuItemComponent implements OnChanges, OnInit {
    }
 
    getOpen(routerActive: boolean): boolean {
-      this.isOpen = routerActive;
+      console.log(this.menu.link.includes(this.activeRoute));
```
Comment (EvaMGC) [inline src/app/layout/menu-item/menu-item.component.ts]: Remove console log

File: src/assets/langs/es.json (hunk 2 above)
Comment (alexrchies) [inline src/assets/langs/es.json]: No string was translated so far. I will keep it without translation until translation will be a real option.

**Pull request merged**